extensions/Gadgets/Gadgets_body.php

```diff
@@ -87,7 +87,10 @@ public static function getPreferences( $user, &$preferences ) {
 			}
 
 			if ( $section !== '' ) {
-				$section = wfMsgExt( "gadget-section-$section", 'parseinline' );
+				// begin wikia change
+				// wfMsg cleanup
+				$section = wfMessage( "gadget-section-$section" )->parse();
```

Comment:
We should consider either adding separate use for each possible value of `$section`, or at least list the possible values in comments - helps when searching for usage of messages